html/changelogs/fethas.yml

```diff
@@ -0,0 +1,13 @@
+# Your name. Remove the quotation mark and put in your name when copy+pasting the example changelog.
+author: Fethas
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Due to Archmage Steve leaving the nya-cromantic stone near the microwave, the true power of the stone has been unleashed. Subjects of the stone are not only ressurected..but ressurected as catgirls..We fired steve."
```

Comment:
if we are going grammar nazi the `.`s should be in threes and have spaces after them